torchvision/prototype/models/efficientnet.py

```diff
@@ -94,7 +94,17 @@ class EfficientNet_B1_Weights(WeightsEnum):
             "acc@5": 94.186,
         },
     )
-    default = ImageNet1K_V1
+    ImageNet1K_V2 = Weights(
+        url="https://download.pytorch.org/models/efficientnet_b1-c2e40cbf.pth",
+        transforms=partial(ImageNetEval, crop_size=224, resize_size=232, interpolation=InterpolationMode.BILINEAR),
+        meta={
+            **_COMMON_META,
+            "recipe": "https://github.com/pytorch/vision/issues/3995#new-recipe-with-lr-wd-crop-tuning",
```

Comment:
Don't forget to update the stats and recipe once you are done here.

Reply:
@prabhat00155 This still needs to be updated. Could you please add the new command and accuracy on the linked issue?